.clang-format

```diff
@@ -0,0 +1,77 @@
+---
```

Comment:
well this is elaborate :-) Is this your personal preference or from some organization?

Reply:
It's what I like within the abilities of clang format 🙂